plugins/thread.rb

```diff
@@ -1,97 +1,71 @@
-#
-# $Id$
-# $Revision$
-#
-
 module Msf
+  class Plugin::ThreadTest < Msf::Plugin
+    class ConsoleCommandDispatcher
+      include Msf::Ui::Console::CommandDispatcher
 
-###
-#
-# This class illustrates a sample plugin.  Plugins can change the behavior of
-# the framework by adding new features, new user interface commands, or
-# through any other arbitrary means.  They are designed to have a very loose
-# definition in order to make them as useful as possible.
-#
-###
```

Comment:
I think we want to leave these comments in still; I'm guessing this is meant to be a sample plugin? 🤔 

Edit: Nope, just left over cruft from the actual sample over here https://github.com/rapid7/metasploit-framework/blob/1b20db8900a5534c7e204be6e2e765ea537d2a32/plugins/sample.rb

Reply:
Yeah the comments are redundant.